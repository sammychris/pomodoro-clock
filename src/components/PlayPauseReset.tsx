import React from "react";

const PlayPauseReset = ({
  start,
  defaultValue,
  sessionOperation,
  breakOperation,
  reset,
}) => {
  return (
    <div id="play-reset">
      <div
        id="start-pause"
        onClick={defaultValue ? sessionOperation : breakOperation}
      >
        <i
          id="start_stop"
          className={start ? "fa fa-pause fa-2x" : "fa fa-play fa-2x"}
        ></i>
      </div>
      <div id="restart" onClick={reset}>
        <i id="reset" className="fa fa-refresh fa-2x"></i>
      </div>
    </div>
  );
};

export default PlayPauseReset;
