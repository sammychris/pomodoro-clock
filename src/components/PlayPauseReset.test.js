import { shallow } from "enzyme";
import PlayPauseReset from "./PlayPauseReset";

const wrapper = shallow(<PlayPauseReset />);

it("expect to render play-pause reset component", () => {
  expect(wrapper.length).toEqual(1);
});

it("expect to take a snap shot of play-pause reset component", () => {
  expect(wrapper).toMatchSnapshot();
});
