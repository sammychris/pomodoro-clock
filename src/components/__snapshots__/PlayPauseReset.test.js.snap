// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to take a snap shot of play-pause reset component 1`] = `ShallowWrapper {}`;
